style.css

```diff
@@ -0,0 +1,297 @@
+* {
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
+}
+
+body {
+  background-color: #f7f7f9;
+}
+
+.background-img {
+  background-image: url(./img/mobile-background.png);
```

Comment:
- [ ] [REQUIRED] All your images are not showing up because you have not added the raw image files to your project. I see that should have a folder called img that has all these images but maybe you forgot to add it. KIndly fix this.